# 统计学：决策的科学项目说明
![image.png](stroopImage.png)
### 背景信息

在一个 Stroop （斯特鲁普）任务中，参与者得到了一列文字，每个文字都用一种油墨颜色展示。参与者的任务是将文字的打印颜色大声说出来。这项任务有两个条件：一致文字条件，和不一致文字条件。在一致文字条件中，显示的文字是与它们的打印颜色匹配的颜色词，如“红色”、“蓝色”。在不一致文字条件中，显示的文字是与它们的打印颜色不匹配的颜色词，如“紫色”、“橙色”。在每个情况中，我们将计量说出同等大小的列表中的墨色名称的时间。每位参与者必须全部完成并记录每种条件下使用的时间。

### 1.我们的自变量是什么？因变量是什么？     

  **自变量**：文字的颜色和文字内容，一致与不一致     
  **因变量**：参与者说出墨色名称的时间

### 2.此任务的适当假设集是什么？你需要以文字和数学符号方式对假设集中的零假设和对立假设加以说明，并对数学符号进行定义。你想执行什么类型的统计检验？为你的选择提供正当理由（比如，为何该实验满足你所选统计检验的前置条件）。      


**适当假设集**：参与者说出墨色的颜色和文字一致或不一致不相关。  
**零假设**： 参与者说出墨色与文字一致不相关    
**对立假设**： 参与者说出墨色与文字颜色一致相关，相应数学符号表示为：

$H_0:\mu_1-\mu_2=0$    

$H_A:\mu_1-\mu_2\neq0$    

其中 $\mu_1$ 表示文字与文字颜色一致时，参与者说出文字颜色的平均时间， $\mu_2$ 表示文字与颜色不同时，参与者说出文字颜色的平均时间。

我想执行单尾检验，因为基于目前的背景信息得出，文字的墨色会与文字内容不一致，会增加参与者读出文字墨色的时间。空口无凭，根据提供的表格计算出平均时间表格如下, 可以从最后一列平均值中看出，文字与墨色颜色一致的平均时间要比不一致的时间少很多。



    

In [9]:
import numpy as np
import pandas as pd

# 数据可视化代码
from IPython.display import display
%matplotlib inline

# 加载数据集
in_file = 'stroopdata.csv'
full_data = pd.read_csv(in_file)

# 显示数据列表
display(full_data)

Congruent       Incongruent
0           12.079            19.278
1           16.791            18.741
2            9.564            21.214
3             8.63            15.687
4           14.669            22.803
5           12.238            20.878
6           14.692            24.572
7            8.987            17.394
8            9.401            20.762
9            14.48            26.282
10          22.328            24.524
11          15.298            18.644
12          15.073             17.51
13          16.929             20.33
14            18.2            35.255
15           12.13            22.158
16          18.495            25.139
17          10.639            20.429
18          11.344            17.425
19          12.369            34.288
20          12.944            23.894
21          14.233             17.96
22           19.71            22.058
23          16.004            21.157
24             NaN               NaN
25  Congruent Avg:  Incongruent Avg:
26       14.051125       22.01591667

现在轮到你自行尝试 Stroop 任务了。前往[此链接](https://faculty.washington.edu/chudler/java/ready.html)，其中包含一个基于 Java 的小程序，专门用于执行 Stroop 任务。记录你收到的任务时间（你无需将时间提交到网站）。[现在下载此数据集](https://s3.cn-north-1.amazonaws.com.cn/static-documents/nd002/stroopdata.csv)，其中包含一些任务参与者的结果。数据集的每行包含一名参与者的表现，第一个数字代表他们的一致任务结果，第二个数字代表不一致任务结果。

### 3.	报告关于此数据集的一些描述性统计。包含至少一个集中趋势测量和至少一个变异测量。

a. 第一组参与者，文字颜色和文字内容一致    
    **均值**为： 14.05    
    **中位数**为：14.3565    
    **众数**为：N/A , 如果时间不保留小数位，众数为12     
   
    
b. 第二组参与者，文字颜色和文字内容不一致     
    **均值**为： 22.02    
    **中位数**为：21.0175    
    **众数**为：N/A , 如果时间不保留小数位，众数为20 

### 4.	提供显示样本数据分布的一个或两个可视化。用一两句话说明你从图中观察到的结果。

**观察结果**：明显文字颜色与文字内容不同的参与者读出墨色颜色的事件远大于一致的参与者，我猜测是人不自觉的读取文字，如果拿这些图片给不认识字的小孩，让孩子说出墨色的颜色，结果应该和第一组用户文字内容和墨色一致的时间相差不多。
![stroop](stroop.png)


### 5.	现在，执行统计测试并报告你的结果。你的置信水平和关键统计值是多少？你是否成功拒绝零假设？对试验任务得出一个结论。结果是否与你的期望一致？

### 6.	可选：你觉得导致所观察到的效应的原因是什么？你是否能想到会取得类似效应的替代或类似任务？进行一些调查研究将有助于你思考这两个问题！